elastic/hatch.toml

```
@@ -24,6 +24,7 @@ matrix.version.env-vars = [
     { key = "ELASTIC_IMAGE", value = "7.7.0", if = ["7.7"] },
     # EOL 2021-12-09
     { key = "ELASTIC_IMAGE", value = "7.9.0", if = ["7.9"] },
+    { key = "ELASTIC_IMAGE", value = "7.10.0", if = ["7.10"] },
```

nit 

```suggestion
    { key = "ELASTIC_IMAGE", value = "7.10.2", if = ["7.10"] },
```

Any reason to not use the latest fix version?

---

None, I think I just copy/pasted `7.9.0` :)